src/System.Management.Automation/FormatAndOutput/common/PSStyle.cs

```diff
@@ -652,6 +652,16 @@ public FileInfoFormatting()
         /// </summary>
         public string Bold { get; } = "\x1b[1m";
 
+        /// <summary>
+        /// Gets value to turn off dim.
+        /// </summary>
+        public string DimOff { get; } = "\x1b[22m";
```

Comment:
The corresponding formatting code will also need to be updated.

Reply:
Good catch!